Translated Software Carpentry version 5.2 and generated various ebook formats in Korean.

statkclee opened the pull request:
Translated Software Carpentry version 5.2 in Korean.

-. Translated the full version of Software Carpentry version 5.2.
-. Generated various ebook formats in Korean.
-. Korean CC license enforcement from CC-BY to CC-BY-SA only for Korean contents, but the other licenses including software are the same.
-. Identified ebook generation software did not support CJK languages
  : parser did not parse Korean characters such that links were broken.

Besides, I'd like to know how to generate various book formats from the current developing lessons. 

Thank you.

r-gaia-cs commented on blog/2015/04/korean-translation-of-software-carpentry-ver-5-2.html:
```diff
@@ -0,0 +1,34 @@
+
+<p> As the starting point of Software Carpentry in Korea, we started translating Software Carpentry lessons last October. <a href="http://software-carpentry.org/blog/2014/11/korean-translation.html">Initial draft version</a> has been released last September with the Greg and Raniere helps. The <a href="http://software-carpentry.org/blog/2014/11/korean-translation.html">first Software Carpentry</a ...
+
+<p> After the fist workshop in Korea, we translated the remaining pieces and attempted to generate various book formats. Unfortunately, there were some issues mostly related to CJK problems. Still there are some issues but learners who can understand Korean can assess the Software Carpentry lessons and related materials through various formats.</p>
```
Minor type here: "After the fist workshop" -> "After the first workshop"

I would replace "generate various book formats." with "generate various ebook formats.".

Again, can you replace "CJK problems" with "problems due  Chinese, Japanese, and Korean characters". I needed to search on Wikipedia for CJK.

r-gaia-cs commented on blog/2015/04/korean-translation-of-software-carpentry-ver-5-2.html:
```diff
@@ -0,0 +1,34 @@
+category: ["Translation", "Version 5.2", "Korea", "Book"]
+---
+<!-- start excerpt -->
+<p> Based on Software Carpentry version 5.2, we translated <a href="http://www.xwmooc.net/swc/">all the lessons</a> in the form of pdf, html, mobi, epub, and azw3. It took almost 6 months: the first half spent on mostly lessons, and the second half spent on remainings. While translating and building various ebook formats, the original book production software of Software Carpentry seemed not to h ...
```
Can you replace "CJK problems" with "problems due  Chinese, Japanese, and Korean characters". I needed to search on Wikipedia for CJK.

r-gaia-cs commented:
@statkclee Thanks for the post and the work with the translation.

The post looks good. It just need a few corrections.

I have change in my machine for EPUB and PDF that I didn't send as a pull request because I still need to fix small problems. I think that I will send it because a pull request is better than nothing. =)

About the CJK, this is due Pandoc? If yes, could you report this to the developers? If you need help please contact me since I already did it a few times.

And I looked at http://www.xwmooc.net/swc/ and at the footnote we have  "Software Carpnery" and it should be "Software Carpentry" as in the header.

statkclee commented:
@r-gaia-cs Thank you for comments. I fixed xwmooc website typo first. I changed words which you recommended including CJK and ebook. I need more study on Pandoc issues. Next I am going to build another software contents based on Software Carpentry platform, so that I can get back to you later.
